layers/meta-resin-intel/recipes-bsp/grub/grub-efi_%.bbappend

```diff
@@ -1,12 +1,12 @@
-FILESEXTRAPATHS_append_intel-corei7-64 := ":${THISDIR}/files"
+FILESEXTRAPATHS_append := ":${THISDIR}/files"
```

Comment:
Why did we remove the machine specific extension? There are many cases around where these were removed. There is no comment in `git log` about this as well.

Reply:
It was obsolete, I can add a note in the git log.

Reply:
@telphan what is obsolete?

Reply:
The machine override, it is not needed as there is only one machine supported.